internal/website/content/pages/blob.md

```diff
@@ -22,6 +22,24 @@ Top-level package documentation: https://godoc.org/gocloud.dev/blob
 * [File-backed local blob](https://godoc.org/gocloud.dev/blob/fileblob) - local
   blob implementation using the file system
 
+The blob package can also interact with most any server that implements the AWS
+S3 HTTP API, like [Minio][], [Ceph][], or [SeaweedFS][]. You can change the
+endpoint used in an S3 URL using query parameters like so:
```

Comment:
Is there a way to do it without URLs?

Reply:
Yes, all of the query parameters correlate 1:1 with `aws.Config` fields. I could rewrite the sample in terms of creating an AWS session, but it would be a bit longer. Do you think it helps clarity?

Reply:
No, just mention that the same can be done with `aws.Config`. Your text gives the impression that you can only do this with URLs.

Reply:
Done.